src/translations.js

```diff
@@ -194,6 +203,7 @@ const repoCardLocales = {
     cs: "Archivováno",
     de: "Archiviert",
     en: "Archived",
+    ml: "ശേഖരിച്ചത്",
```

Comment:
I don't speak Malayalam but google translates gives me `ആർക്കൈവ് ചെയ്തു`. It says `ശേഖരിച്ചത്` means Collected. Can you please double-check this?

Reply:
Ok